cli/src/services/session.service.ts

```diff
@@ -1,31 +1,40 @@
-import fs from 'node:fs';
-import yaml from 'yaml';
+import { existsSync } from 'fs';
```

Comment:
I probably wouldn't remove the `node:` prefix here
```suggestion
import { existsSync } from 'node:fs';
```

Reply:
True, true. Do you know if there is a linting rule or some way to prefer the prefix? The code base is quite inconsistent.

Reply:
I created #6751 as a proof of concept for this (and most other unicorn rules)